src/main/java/seedu/address/model/ModelManager.java

```diff
@@ -22,6 +25,8 @@ public class ModelManager implements Model {
     private final AddressBook addressBook;
     private final UserPrefs userPrefs;
     private final FilteredList<Person> filteredPersons;
+    private final HistoryCommandList historyCommandlist;
```

Comment:
```suggestion
    private final HistoryCommandList historyCommandList;
```

nit: variable naming